README.md

```diff
@@ -39,14 +39,31 @@ To call an endpoint you must create a `Client` object.
 ```python
 from plaid import Client
 
-
 # Available environments are 'sandbox', 'development', and 'production'.
 client = Client(client_id='***', secret='***', public_key='***', environment='sandbox')
 ```
 
 Each endpoint returns a dictionary which contains the parsed JSON from the
 HTTP response.
 
+### Versioning
+
+You can specify the Plaid API version you wish to use when initializing `plaid-python`.
```

Comment:
should `plaid-python` be `plaid`? `plaid-python` is the repo name, not package name

Reply:
⚡️